tests/components/blebox/test_light.py

```diff
@@ -425,6 +453,66 @@ def apply_color(value, color_value):
     assert state.attributes[ATTR_RGBW_COLOR] == (0xC1, 0xD2, 0xF3, 0xC7)
 
 
+# note: it is a good idea to test wide range of values including those
+# outside of the operating range to make sure that bounds are properly
+# corrected
+@pytest.mark.parametrize("temp_requested", [1, 100, 150, 200, 300, 400])
+async def test_wlightbox_on_temp(
+    wlightbox_ctx2, hass: HomeAssistant, temp_requested: int
```

Comment:
Please make hass the first parameter (I see that this is frequent in this file, we should fix that in a followup

Reply:
Fixed, I will update the rest of tests in followup PR and also fix other style issues as I saw things like redundant parentheses around tuple returns and so on.